EventAggregator.Net.SampleUsage/Samples/AsyncSample.cs

```diff
@@ -4,14 +4,15 @@
 
 namespace EventAggregatorNet.SampleUsage.Samples
 {
+#if ASYNC
     public class AsyncSample
     {
         public static void Run()
         {
             var config = new EventAggregator.Config
             {
                 // Make the marshaler run in the background thread
-                DefaultThreadMarshaler = action => Task.Factory.StartNew(action),
+                DefaultThreadAsyncMarshaler = async action => await Task.Factory.StartNew(action),

```

Comment:
I think this should be renamed back and the `marshal` param removed from the async send message overloads